userguide/tn_vcenter.rst

```diff
@@ -42,7 +42,7 @@ shown in
 
 .. _vcenter_plugin_config_fig:
 
-.. figure:: images/tn_vcenter-plugin-config.png
+.. figure:: images/tn_vcenter-plugin-config_a.png
```

Comment:
Let's not rename this image, just replace it with the same name.  Yes, we need to update the process in ix-docs.